wai-aria: Add new test and update expectations for existing test

joanmarie opened the pull request:
The test to verify that the aria-colspan value was ignored as a result
of there being a colspan property on the td element also asserted that
the td element's value (3) was exposed. Because the rendered table only
has one column, the test as written is not reliable.

Limit the assertions of the existing test to verifying that the ARIA
value is ignored. Add a new test with three rendered columns to verify
the td element's value gets exposed instead.

<!-- Reviewable:start -->

<!-- Reviewable:end -->

joanmarie committed:
wai-aria: Add new test and update expectations for existing test

The test to verify that the aria-colspan value was ignored as a result
of there being a colspan property on the td element also asserted that
the td element's value (3) was exposed. Because the rendered table only
has one column, the test as written is not reliable.

Limit the assertions of the existing test to verifying that the ARIA
value is ignored. Add a new test with three rendered columns to verify
the td element's value gets exposed instead.

w3c-bots commented:
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/232351059)

# Firefox (nightly) #
Testing web-platform-tests at revision 777741c86ab95070086d699c7dda762c5dc50def
Using browser at version BuildID 20170514100406; SourceStamp 73b3fc64525b6816842c737e104ef2ac5482d217
Starting 10 test iterations
No tests run.

w3c-bots commented:
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/232351065)

# Chrome (unstable) #
Testing web-platform-tests at revision 777741c86ab95070086d699c7dda762c5dc50def

joanmarie committed:
wai-aria: New and updated tests related to aria-rowspan and aria-colspan

The tests to verify that the aria-{row,col}span value was ignored as a
result of there being a native host-language property set on the element
also asserted that the native host-language property's value was exposed.
Because the rendered tables had fewer {rows,columns} than the specified
spans, these tests are not reliable as written.

Therefore, limit the assertions of the existing tests to verifying that
the ARIA value is ignored. Add new tests in which the host-language
property's stated span is not less than the number of cells rendered.
